spec/models/weather_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe 'weather info' do
+  it 'returns weather info for a trail' do
+    
+    weather = Weather.weather_info(40.13,-103.01)
+
+    expect(weather).to respond_to(:summary)
+    expect(weather).to respond_to(:temp)  
+    expect(weather).to respond_to(:wind)
+  end
+end
```

Comment:
@Mcents Do we want an additional tes to make sure a new instance of weather so we know the poro initializes? Something like...
```it 'initializes with valid attrs` do
    valid_attrs = {
      weather: "Something",
      temperature_string: "Something Else",
      wind_mph: 4
   }

   weather = Weather.new(valid_attrs)

   expect(weather.summary).to eq("Something")
   expect(weather.temperature).to eq("Something Else")
   expect(weather.wind).to eq(4)
end
```
Or something like that just to show they format correctly?